apps/native-component-list/src/screens/WebBrowser/OpenBrowserAsyncDemo.tsx

```diff
@@ -0,0 +1,110 @@
+import * as WebBrowser from 'expo-web-browser';
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import Button from 'src/components/Button';
+import Configurator, { ConfiguratorChoiceType } from 'src/components/Configurator';
+import HeadingText from 'src/components/HeadingText';
+import MonoText from 'src/components/MonoText';
+import { Colors } from 'src/constants';
+
+const url = 'https://blog.expo.dev/expo-sdk-44-4c4b8306584a';
+
+const openBrowserConfigurationChoices: ConfiguratorChoiceType[] = [
+  {
+    name: 'toolbarColor',
+    title: `Use toolbarColor (${Colors.tintColor})`,
+    initial: false,
+    resolve: (checked) => (checked ? Colors.tintColor : undefined),
+  },
+  {
+    name: 'secondaryToolbarColor',
+    title: `Use secondaryToolbarColor (${Colors.highlightColor})`,
+    platforms: ['android'],
+    initial: false,
+    resolve: (checked) => (checked ? Colors.highlightColor : undefined),
+  },
+  {
+    name: 'controlsColor',
+    title: `Use controlsColor (${Colors.headerTitle})`,
+    initial: false,
+    resolve: (checked) => (checked ? Colors.headerTitle : undefined),
+  },
+  {
+    name: 'showTitle',
+    title: 'Show title',
+    platforms: ['android'],
+    initial: false,
+  },
+  {
+    name: 'showInRecents',
+    title: 'Show in recents',
+    platforms: ['android'],
+    initial: false,
+  },
+  {
+    name: 'enableBarCollapsing',
+    title: 'Enable bar collapsing',
+    initial: false,
+  },
+  {
+    name: 'readerMode',
+    title: 'Reader mode',
+    platforms: ['ios'],
+    initial: false,
+  },
+  {
+    name: 'enableDefaultShareMenuItem',
+    title: 'Enable default share menu item',
+    platforms: ['android'],
+    initial: false,
+  },
+];
+
+export default function OpenBrowserAsyncDemo() {
+  const [openOptions, setOpenOptions] = React.useState({});
+  const openBrowser = React.useCallback(async () => {
+    const result = await WebBrowser.openBrowserAsync(url, openOptions);
+    alert(JSON.stringify(result, null, 2));
+  }, [openOptions]);
+
+  const openAndDismiss = React.useCallback(() => {
+    WebBrowser.openBrowserAsync(url, openOptions);
+    WebBrowser.dismissBrowser();
+  }, [openOptions]);
+
+  const openTwice = React.useCallback(async () => {
+    WebBrowser.openBrowserAsync(url, openOptions);
+    const result = await WebBrowser.openBrowserAsync(url, openOptions);
+    if (result.type === WebBrowser.WebBrowserResultType.LOCKED) {
+      alert('Opening the second browser was properly locked');
```

Comment:
I would rather not use `alerts`, but rather something less invasive, like red box with auto-hiding timeout? 🤔